cookiecutter/generate.py

```diff
@@ -40,7 +41,16 @@ def generate_context(context_file='cookiecutter.json', default_context=None,
     context = {}
 
     file_handle = open(context_file)
-    obj = json.load(file_handle, object_pairs_hook=OrderedDict)
+    try:
+        obj = json.load(file_handle, object_pairs_hook=OrderedDict)
+    except ValueError, e:
```

Comment:
This will fail on Python 3. Just switch to `except ValueError as e` and it will work fine. :-)